src/components/App.js

```diff
@@ -7,31 +7,34 @@ import { connect } from 'react-redux'
 import BetaBanner from './BetaBanner'
 import BetaModal from './BetaModal'
 import Disclaimer from './Disclaimer'
+import Feedback from './Feedback'
 import Footer from './Footer'
 import Glossary from './Glossary'
 import Header from './Header'
-import { hideModal } from '../actions/modal'
 
 const App = ({ appState, children, dispatch, location }) => (
   <div className="site">
     <Disclaimer />
-    <BetaBanner />
+    <BetaBanner dispatch={dispatch} />
```

Comment:
I like that and I don't like it. I like it because it reduces the seemingly repetitive `dispatch` props in the `<App />` but I don't like it because I'm not sure the app should have that much logic in it. It is nice that each component pulls in the actions it needs so that all of the "stock" behavior of a component is contained within that file.

However, we do have a pattern elsewhere of using a passed in prop function to handle the change, so I could that being useful. In this case, since the `<BetaBanner />` only has one action, it might make more sense to use `onClose`. I'll change up the components in `<App /`> that only use one or two actions and see if it feels nicer.